### PR TITLE
Implement peer DAS proposer RPC

### PR DESCRIPTION
**Pull request by nalepae**
**What type of PR is this?**
Feature

A lot of tests have still to be implemented. (Tagged with `TODO: Add tests`)

### CODE REVIEW

File: beacon-chain/sync/subscriber_data_column_sidecar.go [hunk 1]
```diff
@@ -6,27 +6,26 @@ import (
...
 	"github.com/prysmaticlabs/prysm/v5/beacon-chain/core/feed"
 	opfeed "github.com/prysmaticlabs/prysm/v5/beacon-chain/core/feed/operation"
-	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/blocks"
 	"google.golang.org/protobuf/proto"
 )
 
 func (s *Service) dataColumnSubscriber(ctx context.Context, msg proto.Message) error {
-	b, ok := msg.(*ethpb.DataColumnSidecar)
+	dc, ok := msg.(blocks.VerifiedRODataColumn)
```
Comment (nisdas) [inline beacon-chain/sync/subscriber_data_column_sidecar.go:14]: this is going to fail, our validation pipelines will pass in a `*ethpb.DataColumnSidecar`. You will need to convert the sidecar to a read only object at the end of the pipeline

Comment (nalepae) [reply beacon-chain/sync/subscriber_data_column_sidecar.go:14]: Fixed in 708be79ee3eafc204cd14d88568b7857af174261.

File: beacon-chain/rpc/prysm/v1alpha1/validator/BUILD.bazel [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# gazelle:ignore
```
Comment (nisdas) [inline beacon-chain/rpc/prysm/v1alpha1/validator/BUILD.bazel:1]: why ? this would ignore the whole file

**Review by nisdas — APPROVED** [review]

File: beacon-chain/rpc/prysm/v1alpha1/validator/BUILD.bazel (hunk 2 above)
Comment (nalepae) [reply beacon-chain/rpc/prysm/v1alpha1/validator/BUILD.bazel:1]: The presence of this tag makes gazelle ignore the whole file, and not only the part just bellow this tag.
As a consequence, it's better to put this tag at the top of the file, so the reader immediately knows gazelle will ignore it.

**Pull request merged**